runtime/verbose/verbose.c

```diff
@@ -633,6 +641,12 @@ parseVerboseArgument(char* options, J9VerboseSettings* verboseOptions, char** er
 			verboseOptions->relocations = VERBOSE_SETTINGS_SET;
 		} else if(strcmp(options, OPT_ROMCLASS)==0) {
 			verboseOptions->romclass = VERBOSE_SETTINGS_SET;
+#if (JAVA_SPEC_VERSION >= 11)
+		} else if(strcmp(options, OPT_NOMODULE)==0) {
+			verboseOptions->module = (VERBOSE_SETTINGS_SET == verboseOptions->module) ? VERBOSE_SETTINGS_IGNORE: VERBOSE_SETTINGS_CLEAR;
```

Comment:
Is setting VERBOSE_SETTINGS_IGNORE or VERBOSE_SETTINGS_CLEAR still needed, I don't see these used.

Reply:
The reason it's needed for `nogc` and `nocvlass` is to support enabling/disabling verbose output at runtime.
https://docs.oracle.com/javase/8/docs/api/java/lang/management/ClassLoadingMXBean.html#setVerbose-boolean-
https://docs.oracle.com/en/java/javase/11/docs/specs/jvmti.html#SetVerboseFlag

Reply:
Thanks I removed that line. The OPT_MODULE input is there so that `verbose:nomodule` is recognized.

Reply:
I just meant the line should be simplified, like follows. I don't think your change is correct, the test behavior shouldn't change.
`verboseOptions->module = VERBOSE_SETTINGS_IGNORE;`

Reply:
hotspot doesn't seem to do anything to disable the module output when both `verbose:module` and `verbose:nomodule` are set. Thats likely a mistake though.

Reply:
I'll add that line back in the simplified state.

Reply:
`-verbose:module,nomodule` does do the expected on Hotspot (no module output). I agree `-verbose:module -verbose:nomodule` showing module output seems like a Hotspot bug.

Reply:
Updated.